### ISSUE TITLE
Format date in Forms

### ISSUE THREAD

**Issue by krsmga** [z-song/laravel-admin#1727]
I'm trying to use the format () method to display the Date field in Brazilian format. But it does not work. The field appears empty when editing the record and this only happens when I define the format () method.

$form->datetime('created_at', 'Criado em:')->format('DD/MM/YYYY HH:mm:ss');

**Comment by NickCoolii** [comment]
@krsmga have same problem.
I'm tried to make it work with "[Accessors & Mutators]" (https://laravel.com/docs/5.5/eloquent-mutators), but no succeed.
As far as I know format() makes problem with wrong datetime format which provides into sql query.

**Comment by diogolast** [comment]
I am using Accessors & Mutators  and is working well.

```
    public function getDataNascimentoAttribute($value)
    {
        return \Carbon\Carbon::parse($value)->format('d/m/Y');
    }

    public function setDataNascimentoAttribute($value)
    {
        $this->attributes['data_nascimento'] = \Carbon\Carbon::createFromFormat('d/m/Y', $value)->format('Y-m-d');
    }
```

And the form I put this

$form->date('data_nascimento', 'Data de Nascimento*')->rules('required')->format('DD/MM/YYYY');

**Comment by lordjack** [comment]
@diogolast worked with me, thanks!

**Comment by NickCoolii** [comment]
@diogolast don't you have any troubles with date filter at grid using Accessors & Mutators?

**Comment by SergAndrs** [comment]
Greetings.
I have the same issue with date. Here is my accessor

`public function getStartDateAttribute($value)
{

        return Carbon::parse($value)->format('d.m.Y');
}`

When I displayed it in form it returns current date, whenever I tried any format for date.
`$form->date('start_date', 'Start date')->format('DD.MM.YYYY')`
Also tried to rewrite my accessor as mentioned @diogolast but still return current date.

Would be appreciate for any help
